core/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "@umaprotocol/core",
+  "version": "1.0.0",
+  "description": "UMA smart contracts and unit tests",
+  "private": true,
+  "dependencies": {
+    "@openzeppelin/contracts": "3.0.0",
+    "minimist": "^1.2.0",
+    "inquirer": "^7.0.4",
+    "dotenv": "^6.2.0",
+    "winston": "^3.2.1",
+    "chalk-pipe": "^3.0.0",
+    "ora": "^4.0.3",
+    "moment": "^2.24.0",
+    "bignumber.js": "^8.0.1",
+    "@sendgrid/mail": "^6.4.0",
```

Comment:
@sendgrid, node-fetch, and gmail-send are only used by the AVS (Voting.js) script, is this script deprecated now?

Reply:
It's a good question. I think we should probably bring this up with the broader team to make sure we don't want to pursue this strategy in the future.